src/bindist/dosemu-downloaddos

```diff
@@ -126,12 +126,12 @@ def download_dos(dos_flavour, destination):
 
     if dos_flavour == 'freedos12':
         download_FREEDOS_KERNEL_COMMAND(FREEDOS12_URL, destination)
-    if dos_flavour == 'freedos12userspace':
+    if dos_flavour == 'freedos12userspace' or dos_flavour == dos_flavour == 'freedos12':
```

Comment:
Perhaps?
```python
if dos_flavour in ['freedos12userspace', 'freedos12']:
```

Reply:
Thanks! Fixed that! I am not so used to Python yet.